index.d.ts

```diff
@@ -42,6 +42,18 @@ declare module 'vend-number' {
     EUCLID = 9
   }
 
+  export const ROUNDING_MODES = {
```

Comment:
Could also change the `ROUNDING_MODES` type inside the class to by `typeof ROUNDING_MODES`, to reduce duplication? I think that works?